apps/mdn/mdn-aws/k8s/regions/oregon/stage.sh

```diff
@@ -110,6 +110,7 @@ export KUMA_CELERYD_MAX_TASKS_PER_CHILD=0
 export KUMA_CSP_ENABLE_MIDDLEWARE=True
 export KUMA_CSP_REPORT_ENABLE=True
 export KUMA_CSP_REPORT_ONLY=True
+export KUMA_CSP_REPORT_URI=https://sentry.prod.mozaws.net/api/72/security/?sentry_key=25e652a045b642dfaa310e92e800058a
```

Comment:
``api/72`` is sentry's ``mdn-stage`` app. and ``api/73`` is stage's ``mdn-prod``, so we get stage vs prod for free. However, while we had two environments, it may have been nice to distinguish them. Maybe ``environment=oregon`` or ``environment=oregon:stage``? I could imagine demo environments would report to the mdn-stage endpoint, so maybe ``environment=stage`` would work... I'll chew on it, see what seems best before the push. We'd potentially want the same environment setting from Django.

Reply:
I'm going to try ``<region>:<type>`` as an environment pattern, so this will be ``oregon:stage``, ``oregon:stage.mm``, etc. We'd probably want demos to report to the ``mdn-stage`` app in the future, this will help expand to those.

Reply:
I didn't know about the `stage` and `prod` links to `api/72` and `api/73`, and I like your idea of using it for preparing fo rthe possible use of demos in the future. Thanks @jwhitlock! I